tests/test_measurement_model.py

```diff
@@ -0,0 +1,49 @@
+"""Testing the measurement model class."""
+
+import numpy as np
+import polars as pl
+import pytest
+
+from wristpy.core.models import Measurement
+
+
+@pytest.fixture
+def create_mock_measurement() -> Measurement:
+    """Create a mock measurement instance."""
+
+    def _create_mock_measurement(
+        measurements: np.ndarray, time: np.ndarray
+    ) -> Measurement:
+        time = time * 1000000
+        time_series = pl.from_epoch(pl.Series(time), time_unit="us")
+        return Measurement(measurements=measurements, time=time_series)
+
+    return _create_mock_measurement
+
+
+def test_measurement_model_time_type() -> None:
+    """Test the error when time is not a datetime series."""
+    with pytest.raises(ValueError):
+        time = pl.Series([1, 2, 3])
```

Comment:
Move the `time =` outside the raises; the raises should only occur in the next line and we want to be sure of this.